samples/snippets/src/main/java/com/example/speech/Recognize.java

```diff
@@ -918,4 +918,41 @@ public static void transcribeMultiChannelGcs(String gcsUri) throws Exception {
     }
   }
   // [END speech_transcribe_multichannel_gcs]
+
+  // [START speech_transcribe_with_profanity_filter_gcs]
+  /**
+   * Transcribe a remote audio file with multi-channel recognition
+   *
+   * @param gcsUri the path to the audio file
+   */
+  public static void syncRecognizeWithProfanityFilterGcs(String gcsUri) throws Exception {
```

Comment:
Can you split this sample (and test) into [its own file/class](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#class-structure)? New samples should be in their own files.

Reply:
Sure thing. I added it in the ```samples/snippets/src/main/java/com/example/speech/Recognize.java``` because that's where the rest of the feature-showing samples reside. But I can relocate my new sample, no problem. I plan on adding one more sample next so thanks for advising me on the new rules.

Reply:
@chingor13 applied suggested changes.